dataline-integrations/singer/bigquery/destination/build.gradle

```diff
@@ -1,34 +1,51 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
```

Comment:
Going back on a past comment I did, it is actually really anoying to have imports sprinkled in the code.